package.json

```diff
@@ -29,4 +29,4 @@
     "url": "https://github.com/unlock-protocol/unlock/issues"
   },
   "homepage": "https://github.com/unlock-protocol/unlock#readme"
-}
\ No newline at end of file
+}
```

Comment:
please revert all changes to package.json. You'll probably need to rebase against master.

```bash
git fetch
git rebase origin/master
git push --force
```

If you have conflicts, you'll need to resolve those. If you're uncomfortable doing rebase, you can also do a merge commit, github has a way to do a pull request to your branch from the main one which you can then merge.